### PR TITLE
Update nlp feature extractor to bump protobuf version for vulnerability. Add pip freeze to show packages installed.

### PR DESCRIPTION
**Pull request by kicha0**

### CODE REVIEW

File: responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ channels:
   - anaconda
 dependencies:
   - python=3.8
+  - pip
```
Comment (tongyu-microsoft) [inline responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:8]: is this needed?

Comment (kicha0) [reply responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:8]: we were pinning pip to an older version before that I think contributed to conda grabbing some older versions of base python packages, along with the conda channels.

Right now in build log conda actually complain about missing pip:

Retrieving notices: ...working... done
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Collecting package metadata (repodata.json): ...working...

**Review by tongyu-microsoft — DISMISSED** [review]

**Review by imatiach-msft — APPROVED** [review]

File: responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ channels:
   - pytorch
 dependencies:
   - python=3.8
+  - pip
```
Comment (vizhur) [inline responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:10]: nit: i'd recommend to pin pip

File: responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml [hunk 3]
```diff
@@ -26,8 +27,9 @@ dependencies:
     - azureml-dataset-runtime
     - azureml-core
     - --extra-index-url https://azuremlsdktestpypi.azureedge.net/sdk-cli-v2
-    - nlp-feature-extractors==0.0.3
-    - https://publictestdatasets.blob.core.windows.net/packages/pypi/responsibleai-text/responsibleai_text-0.0.7-py3-none-any.whl
+    - nlp-feature-extractors==0.1.0
+    - datasets>=2.9.0
+    - https://publictestdatasets.blob.core.windows.net/packages/pypi/responsibleai-text/responsibleai_text-0.0.13-py3-none-any.whl
```
Comment (vizhur) [inline responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:32]: consider to use findlink instead

**Review by vizhur — APPROVED** [review]

File: responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml (hunk 3 above)
Comment (tongyu-microsoft) [reply responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:32]: Created a work item on this, will address in the future as all gates are green and ready to check in.

Comment (tongyu-microsoft) [reply responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:32]: [Task 2202691](https://msdata.visualstudio.com/Vienna/_workitems/edit/2202691): curated image update for findlink, pip

File: responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml (hunk 2 above)
Comment (tongyu-microsoft) [reply responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:10]: [Task 2202691](https://msdata.visualstudio.com/Vienna/_workitems/edit/2202691): curated image update for findlink, pip

**Pull request merged**